tests/e2e/management/test_management.py

```diff
@@ -415,6 +415,8 @@ def test_trash_runs_and_models(self, project, environment):
         run2_id = initialize_container(ContainerType.RUN, project=environment.project)["sys/id"].fetch()
         model1_id = initialize_container(ContainerType.MODEL, project=environment.project)["sys/id"].fetch()
         model2_id = initialize_container(ContainerType.MODEL, project=environment.project)["sys/id"].fetch()
+        # wait for the elasticsearch cache to fill
+        sleep(5)
```

Comment:
@Raalsky what do you think about creating some `eventualy(predicate, timeout)` method with expotentional retries?

Reply:
I've created a task for that https://neptune-labs.atlassian.net/browse/NPT-13399

Reply:
I've already been using `backoff` package in a few integrations tests. Something like:
```python
@backoff.on_exception(backoff.expo, AssertionError, max_value=1, max_time=60)
def test_....():
    ....
```

Reply:
I think that there is something like a constant for the number of seconds to wait.

Reply:
Okay, will do

Reply:
Done